webapp/src/DOMJudgeBundle/Utils/Utils.php

```diff
@@ -1087,10 +1120,17 @@ public static function generatePassword()
     public static function phpini_to_bytes(string $size_str)
     {
         switch (substr($size_str, -1)) {
-            case 'M': case 'm': return (int)$size_str * 1048576;
-            case 'K': case 'k': return (int)$size_str * 1024;
-            case 'G': case 'g': return (int)$size_str * 1073741824;
-            default: return $size_str;
+            case 'M':
+            case 'm':
+                return (int)$size_str * 1048576;
+            case 'K':
+            case 'k':
+                return (int)$size_str * 1024;
+            case 'G':
+            case 'g':
+                return (int)$size_str * 1073741824;
+            default:
+                return $size_str;
```

Comment:
Not cleaner to just strtolower?

Reply:
If removing the nested case on a single line, then `strtolower` is cleaner indeed. Also, if changing this, then I'd prefer to sort by modifier size, so k,m,g (or reverse).

Reply:
Utils changes have been removed.